addons/web_editor/static/src/js/editor/snippets.options.js

```diff
@@ -1813,15 +1813,18 @@ const SelectPagerUserValueWidget = SelectUserValueWidget.extend({
      * @override
      */
     _onClick(ev) {
-        const activeButton = this._getActiveSubWidget();
-        if (activeButton) {
-            const currentPage = this.pages.indexOf(activeButton.el.closest('we-select-page'));
-            if (currentPage !== -1) {
-                this.currentPage = currentPage;
-                this._updatePage();
+        if (!ev.target.closest('.o_we_pager_header')) {
```

Comment:
The opposite form makes more sense, especially when updating stable code:
```javascript
if (ev.target.closest('.o_we_pager_header')) {
    ev.stopPropation();
    return;
}
// ...
return this._super(...arguments);
```

Not only I find that easier to read and I prefer to always finish a method with its _super call not inside an if/else, it also has the advantage of not having to indent the previous code, making git history better and review easier :+1: